end-to-end-test/shared/specUtils.js

```diff
@@ -18,6 +18,12 @@ function waitForCoExpressionTab(timeout) {
     $('//*[@id="coexpressionTabGeneTabs"]').waitForExist(timeout || 20000);
 }
 
+function waitForPatientView(timeout) {
+    $('//*[@id="patientViewPageTabs"]').waitForExist(timeout || 20000);
```

Comment:
can you use #patientViewPageTabs here too?

Reply:
Done.